tests/cleanup/cleanup_test_bundles.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import os
+import unittest
+
+from ..data_store_agent import DataStoreAgent
+
+class CleanupTestBundles(unittest.TestCase):
+    test_bundle_query = {
+      "query": {
+        "bool": {
+          "should": [
+            {
+              "prefix": {
+                "files.project_json.project_core.project_short_name": "prod"
+              }
+            }
+          ],
+          "must_not": [
+            {
+              "match": {
+                "files.analysis_process_json.type.text": "analysis"
```

Comment:
Why exclude analysis bundles? Don't you want to remove them as well?

Reply:
Yes, thanks :)